mappings/net/minecraft/world/gen/structure/RuinedPortalStructure.mapping

```diff
@@ -26,6 +28,12 @@ CLASS net/minecraft/class_5183 net/minecraft/world/gen/structure/RuinedPortalStr
 		ARG 6 box
 		ARG 7 world
 		ARG 8 noiseConfig
+	METHOD method_41682 sampleRandom (Lnet/minecraft/class_2919;F)Z
```

Comment:
What does `sampleRandom` mean? It stands out as a bit off since it reads like unnatural naming. (e.g. BlockPiston vs PistonBlock)

Reply:
It samples the random instance.

Reply:
IMO `sampleRandom` is unneccessarily generic. If I were you, I'd pick a name that's indicative that its sole purpose is to decide whether to place an air pocket.
![image](https://user-images.githubusercontent.com/6429283/221411851-be10354c-f157-409b-87c6-da21add50b87.png)

Reply:
```suggestion
	METHOD method_41682 shouldPlaceAirPocket (Lnet/minecraft/class_2919;F)Z
```